ansible/modules/hashivault/hashivault_azure_secret_engine_config.py

```diff
@@ -105,9 +105,10 @@ def main():
     argspec['client_secret'] = dict(required=False, type='str')
     argspec['environment'] = dict(required=False, type='str', default='AzurePublicCloud')
     argspec['config_file'] = dict(required=False, type='str', default=None)
+    supports_check_mode=True
     required_together=[['subscription_id', 'client_id', 'client_secret', 'tenant_id']]
 
```

Comment:
required_together is actually the 4th arg, so I'm going to change this back.

Reply:
thanks!

Reply:
Connect with me on https://www.linkedin.com/in/terrylhowe/ if you like